frontend/app/templates/components/mapbox/filter-features.hbs

```diff
@@ -0,0 +1 @@
+{{this.filterToFeatures}}
```

Comment:
Hmm is... this is a computed property, yes? Not sure if it's an array of features or something, but seems like some kind of array or object. Anyways, why is it being printed? Is it supposed to be `{{yield}}` for some reason?

I would just make this `{{yield}}` unless you rly need this for some reason.

Reply:
👍 Yah, cleaned it off in my upcoming amend!